### PR TITLE
fix(frontend): auth session ttl specified in hours instead of days. M…

### PR DESCRIPTION
**Pull request by thomasplarsson**
…ore reasonable default 24 hours instead of 30 days.

Fixes: #2680

## Checklist
- [x] The PR conforms to DataHub's [Contributing Guideline](https://github.com/linkedin/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/linkedin/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [x] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable)

### CODE REVIEW

**Comment by thomasplarsson** [comment]
@jjoyce0510 I'm aware that this might change behaviour for existing users, but having a default session lifetime of 24 hours instead of 30 days feels more reasonable. The configuration property was not even exposed in the application.conf leading me to believe that not many deployments will have overridden this.

**Comment by jjoyce0510** [comment]
@thomasplarsson This means users will have to login every day. Why do you think this is more reasonable?

**Comment by thomasplarsson** [comment]
@jjoyce0510  I might be biased here, coming from a regulated industry where allowing such long-lived session is rarely allowed. But changing the default back to 30 days would be perfectly fine for us if we could keep the possibility to control it with hourly granularity. WDYT?

**Comment by jjoyce0510** [comment]
@thomasplarsson I think that makes a lot of sense. thank you!

**Comment by thomasplarsson** [comment]
@jjoyce0510 Not sure if I interpret you correctly but I changed the default timeout back to 30 days (720 hours).

**Comment by thomasplarsson** [comment]
Hi @jjoyce0510 or @shirshanka. Do you have any more comments on this?

File: datahub-frontend/conf/application.conf [hunk 1]
```diff
@@ -137,6 +137,10 @@ auth.jaas.enabled = ${?AUTH_JAAS_ENABLED}
 # auth.jaas.enabled = false
 # auth.oidc.enabled = false # (or simply omit oidc configurations)
 
+# Login session expiration time
+auth.session.ttlInHours = ${?AUTH_SESSION_TTL_HOURS}
```
Comment (jjoyce0510) [inline datahub-frontend/conf/application.conf:141]: Should we keep this commented out by default, like the config immediately above?

Comment (thomasplarsson) [reply datahub-frontend/conf/application.conf]: Fixed.

**Review by jjoyce0510 — APPROVED** [review]
This LGTM. Thanks!

**Review by shirshanka — APPROVED** [review]
LGTM!

**Pull request merged**